src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/UseGenericEventHandlerInstances.cs

```diff
@@ -84,7 +84,9 @@ public override void Initialize(AnalysisContext context)
                     }
 
                     static bool IsDelegateTypeWithInvokeMethod(INamedTypeSymbol namedType) =>
+#pragma warning disable IDE0078 // Use pattern matching - https://github.com/dotnet/roslyn/issues/51691
                         namedType.TypeKind == TypeKind.Delegate && namedType.DelegateInvokeMethod != null;
```

Comment:
❔ Is this not the same?

```suggestion
                        namedType is { TypeKind: TypeKind.Delegate, DelegateInvokeMethod: not null };
```

Reply:
@sharwell There are actually more IDE0078 violations than the ones I suppressed. Some of them are actionable like you suggested, and others doesn't seem to be actionable. I'm thinking of setting the rule to silent for now until the analyzer and codefix bugs are fixed to avoid the many suppressions and manual fixes. Would you be okay with that?

Reply:
Certainly